Add verbose mode to opass

gtback opened the pull request:
To use this, run `opass -v ...` or `opass --verbose`. This will show the
full `pass` and `git-svn` command as they are executed.

gtback committed:
Add verbose mode to opass

To use this, run `opass -v ...` or `opass --verbose`. This will show the
full `pass` and `git-svn` command as they are executed.

kfogel commented on opass:
```diff
@@ -257,8 +257,11 @@ def main():
     try:
-        (opts, args) = getopt.getopt(sys.argv[1:], "ho", ["help", "offline",])
+        (opts, args) = getopt.getopt(sys.argv[1:], "hov",
+                ["help", "offline", "verbose"])
```
Don't forget comma after last item in tuple -- not strictly necessary, but good form and protects against oddball mis-parsings that can happen in edge cases.

gtback replied:
This is a list, not a tuple, so I don't think the same mis-parsings that can happen with a single-item tuple are likely here, but nevertheless I re-added the comma for consistency with the rest of the file.

gtback replied:
Addressed in 04df5a5f17c9388a7c7fb3a79d7b6717f33cfe83

kfogel commented on opass:
```diff
@@ -325,7 +330,10 @@ def main():
-            completed = subprocess.run([pass_pgm, 'git-svn', thing,],
+            args = [pass_pgm, 'git-svn', thing,]
+            if verbose:
+                print(args)
```
Can there be some window dressing around this?  That is, instead of just printing the raw args, print a status message that explains where we are ("About to execute 'pass git-svn {thing}...") and then shows the args, so that the user has some context?

gtback replied:
Addressed in 04df5a5f17c9388a7c7fb3a79d7b6717f33cfe83

kfogel commented on opass:
```diff
@@ -365,7 +373,10 @@ def main():
-            subprocess.run([pass_pgm,] + things, input=content,
+            args = [pass_pgm,] + things
+            if verbose:
+                print(args)
```
Same here.

gtback replied:
Addressed in 04df5a5f17c9388a7c7fb3a79d7b6717f33cfe83

gtback committed:
Address PR feedback

- Add trailing comma to list
- Give context when printing command arguments in verbose mode

gtback commented:
@kfogel Here is what it looks like now:

```
❯ opass -v show test/everyone/hello
About to execute the following 'pass git-svn' command:
['/usr/local/bin/pass', 'git-svn', 'rebase']
About to execute the following 'pass' command:
['/usr/local/bin/pass', 'show', 'test/everyone/hello']
...
```

kfogel commented:
Merged!  Thanks, @gtback.
